app/src/main/java/com/teamagam/gimelgimel/app/map/actions/send/dynamicLayers/EditDynamicLayerViewModel.java

```diff
@@ -167,15 +203,52 @@ public void onNewTabSelection(int newTabResource) {
     }
   }
 
+  public void onEditDescriptionFabClicked() {
+    EditText input = new EditText(mContext);
+    input.setText(mEditedDescription);
+    if (mEditedDynamicEntity != null) {
+      mEditedDescription = mEditedDynamicEntity.getDescription();
+      createEditDescriptionAlertDialog(input, (dialogInterface, i) -> {
+        mEditedDescription = input.getText().toString();
+        sendEntityDescription();
+        mDynamicLayerDetailsFragment.getViewModel().updateDescription(mEditedDescription);
+      }, mContext.getString(R.string.edit_dynamic_layer_entity_alert_title), "");
```

Comment:
Why there is no message here but in the next occurrence there is?

Reply:
fixed